frontendFile/src/main/java/com/virtuslab/gitmachete/frontend/file/highlighting/MacheteAnnotator.java

```diff
@@ -66,6 +69,16 @@ private void processIndentationElement(PsiElement element, AnnotationHolder hold
 
     IndentationParameters indentationParameters = findIndentationParameters(element);
 
+    // Potentially unrelated - it's responsible of changing default TAB key behavior
```

Comment:
Btw the dedent operation (Shift+Tab) still dedents by a single space even if the indent char is set to tab... not sure if there's a separate setting that controls that?